### ISSUE TITLE
org.elasticsearch.index.VersionType$1, $2 and $3 was unintentionally innitialized at build time

### ISSUE THREAD

**Issue by ivangfr** [spring-projects-experimental/spring-native#1227]
Hi, I've updated my [`springboot-elasticsearch`](https://github.com/ivangfr/graalvm-quarkus-micronaut-springboot/tree/master/elasticsearch/springboot-elasticsearch) project to **spring-boot** `2.5.6` and **spring-native** `0.10.5`.

At the end of the native compilation, there is the following exection
```
[INFO]     [creator]     [/layers/paketo-buildpacks_native-image/native-image/com.mycompany.springbootelasticsearch.SpringbootElasticsearchApplication:158]        image:  12,367.76 ms,  5.87 GB
[INFO]     [creator]     [/layers/paketo-buildpacks_native-image/native-image/com.mycompany.springbootelasticsearch.SpringbootElasticsearchApplication:158]        write:   1,229.29 ms,  5.87 GB
[INFO]     [creator]     Error: Classes that should be initialized at run time got initialized during image building:
[INFO]     [creator]      org.elasticsearch.index.VersionType was unintentionally initialized at build time. org.elasticsearch.index.VersionType has been initialized without the native-image initialization instrumentation and the stack trace can't be tracked. Try marking this class for build-time initialization with --initialize-at-build-time=org.elasticsearch.index.VersionType
[INFO]     [creator]     org.elasticsearch.index.VersionType$1 was unintentionally initialized at build time. org.elasticsearch.index.VersionType$1 has been initialized without the native-image initialization instrumentation and the stack trace can't be tracked. Try marking this class for build-time initialization with --initialize-at-build-time=org.elasticsearch.index.VersionType$1
[INFO]     [creator]     org.elasticsearch.index.VersionType$2 was unintentionally initialized at build time. To see why org.elasticsearch.index.VersionType$2 got initialized use --trace-class-initialization=org.elasticsearch.index.VersionType$2
[INFO]     [creator]     org.elasticsearch.index.VersionType$3 was unintentionally initialized at build time. org.elasticsearch.index.VersionType$3 has been initialized without the native-image initialization instrumentation and the stack trace can't be tracked. Try marking this class for build-time initialization with --initialize-at-build-time=org.elasticsearch.index.VersionType$3
[INFO]     [creator]
[INFO]     [creator]     Error: Use -H:+ReportExceptionStackTraces to print stacktrace of underlying exception
[INFO]     [creator]     [/layers/paketo-buildpacks_native-image/native-image/com.mycompany.springbootelasticsearch.SpringbootElasticsearchApplication:158]      [total]: 353,769.01 ms,  5.87 GB
[INFO]     [creator]     # Printing build artifacts to: /layers/paketo-buildpacks_native-image/native-image/com.mycompany.springbootelasticsearch.SpringbootElasticsearchApplication.build_artifacts.txt
[INFO]     [creator]     Error: Image build request failed with exit status 1
[INFO]     [creator]     unable to invoke layer creator
[INFO]     [creator]     unable to contribute native-image layer
[INFO]     [creator]     error running build
[INFO]     [creator]     exit status 1
[INFO]     [creator]     ERROR: failed to build: exit status 1
```

**How to reproduce**
- git clone https://github.com/ivangfr/graalvm-quarkus-micronaut-springboot.git
- cd graalvm-quarkus-micronaut-springboot
- git checkout update-frameworks-version
- cd elasticsearch/springboot-elasticsearch
- ./mvnw clean spring-boot:build-image

**Comment by ivangfr** [comment]
Hi @sdeleuze btw, this issue seems to be resolved in **spring-native** `0.11.0-RC1` and current `0.11.0-SNAPSHOT`

**Comment by sdeleuze** [comment]
Closing it as fixed in the latest version where elasticsearch is supported.
